<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareMemoryViewer_win</class>
 <widget class="QMainWindow" name="ShareMemoryViewer_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10,2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="monsbsViewTab">
       <attribute name="title">
        <string>   遥信量   </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QSpinBox" name="yxFilterEd1">
                 <property name="mouseTracking">
                  <bool>false</bool>
                 </property>
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="minimum">
                  <number>16777216</number>
                 </property>
                 <property name="maximum">
                  <number>33554432</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="yxFilterEd5">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>&gt;= MeasureId &lt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="yxFilterEd2">
                 <property name="minimum">
                  <number>16777216</number>
                 </property>
                 <property name="maximum">
                  <number>33554432</number>
                 </property>
                 <property name="value">
                  <number>16778216</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="yxFilterCb">
                 <property name="text">
                  <string>filter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="yxFilterEd6">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="monsbDataTw"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QSpinBox" name="yxFilterEd1_2">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>33554432</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>&gt;= ChangedIndex &lt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="yxFilterEd2_2">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="yxFilterCb_2">
                 <property name="text">
                  <string>filter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="monsbChangedTw"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ycaddViewTab">
       <attribute name="title">
        <string>   遥测量   </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <widget class="QWidget" name="layoutWidget_2">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QSpinBox" name="ycFilterEd1">
                 <property name="minimum">
                  <number>33554432</number>
                 </property>
                 <property name="maximum">
                  <number>50331648</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>&gt;= MeasureId &lt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="ycFilterEd2">
                 <property name="minimum">
                  <number>33554432</number>
                 </property>
                 <property name="maximum">
                  <number>50331648</number>
                 </property>
                 <property name="value">
                  <number>33555432</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="ycFilterCb">
                 <property name="text">
                  <string>filter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="ycaddDataTw"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget_3">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QSpinBox" name="ycFilterEd1_2">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>50331648</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>&gt;= ChangedIndex &lt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="ycFilterEd2_2">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="ycFilterCb_2">
                 <property name="text">
                  <string>filter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="ycaddChangedTw"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="strawViewTab">
       <attribute name="title">
        <string>   遥文量   </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <widget class="QWidget" name="layoutWidget_4">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QSpinBox" name="ywFilterEd1">
                 <property name="minimum">
                  <number>50331648</number>
                 </property>
                 <property name="maximum">
                  <number>67108864</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>&gt;= MeasureId &lt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="ywFilterEd2">
                 <property name="minimum">
                  <number>50331648</number>
                 </property>
                 <property name="maximum">
                  <number>67108864</number>
                 </property>
                 <property name="value">
                  <number>50332648</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="ywFilterCb">
                 <property name="text">
                  <string>filter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="strawDataTw"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget_5">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QSpinBox" name="ywFilterEd1_2">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>67108864</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>&gt;= ChangedIndex &lt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="ywFilterEd2_2">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="ywFilterCb_2">
                 <property name="text">
                  <string>filter</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="strawChangedTw"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>   控制区   </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>遥控</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_7"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_7">
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="ykTw"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>遥调</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_8"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_8">
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="ytTw"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QWidget" name="widget_10" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>遥文</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_9"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_9">
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="ywTw"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>   文件信息区   </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QWidget" name="widget_12" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QSpinBox" name="fileInfoFilterEd1_2">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>67108864</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>&gt;= ChangedIndex &lt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="fileInfoFilterEd2_2">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="fileInfoFilterCb">
               <property name="text">
                <string>filter</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="fileInfoTw"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>退出</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
